### ISSUE TITLE
v3.21 Pro - Interface crashes when editing Date tags' values.

### ISSUE THREAD

**Issue by gitdocke** [tagspaces/tagspaces#1016]
When I uncheck the "Show Time" checkmark, the interface gets blank.

Tagspaces v3.2.1 Pro, tar.gz portable version
KDE Neon 5.16, (Ubuntu 18.04 LTS)

![Peek 2019-08-21 05-54](https://user-images.githubusercontent.com/22734449/63422883-34a41980-c3d9-11e9-8150-dd9591b57da2.gif)
![Peek 2019-08-21 05-55](https://user-images.githubusercontent.com/22734449/63422898-3b329100-c3d9-11e9-924f-c389bd718457.gif)

**Comment by gitdocke** [comment]
I discovered also, that the interface will crash when I try to edit a regular Date tag, as shown below.

![Peek 2019-08-21 19-17](https://user-images.githubusercontent.com/22734449/63474912-dc0c6500-c448-11e9-9145-35802231c8e7.gif)
----------------------------

Tagspaces v3.2.1 Pro - DEB version
KDE Neon 5.16 (Ubuntu 18.04 LTS)

**Comment by gitdocke** [comment]
I found another interface crash while checking internet for a new version, this time with v3.1.4 Pro. It might be related to this #1016 bug.

Tagspaces v3.1.4 Pro.
KDE Neon 5.16 (Ubuntu 18.04 LTS)

![Peek 2019-09-12 02-26](https://user-images.githubusercontent.com/22734449/64759522-563e7f80-d505-11e9-845e-40555aaabce1.gif)

**Comment by uggrock** [comment]
Thanks for reporting the last issue was just fixed for the next release.

**Comment by uggrock** [comment]
@gitdocke Is this issue still reproducible for you in 3.2.3?

**Comment by gitdocke** [comment]
@uggrock

I did the following test.
Using v3.2.1 Pro, I tagged 4 files:
    File 1: tagged with "Year"
    File 2: tagged with "Today"
    File 3: tagged with "Custom Date"
    File 4: tagged with "Custom Date" and option "Range" checkmarked.
As shown below.

![Peek 2019-10-07 02-46](https://user-images.githubusercontent.com/22734449/66291040-c6df7e80-e8ae-11e9-9a88-079472c8da45.gif)


Then using "tagspaces-linux-x86_64-3.2.3.AppImage" edited those tags without incident, as shown in the below video.

![Peek 2019-10-07 02-51](https://user-images.githubusercontent.com/22734449/66291064-d3fc6d80-e8ae-11e9-8542-1045ac2a3e50.gif)

Thank you for your attention.

---------

Tagspaces v3.2.1 Pro.
KDE Neon 5.16 (Ubuntu 18.04 LTS)
